examples/custompopups/main.qml

```diff
@@ -0,0 +1,110 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.WebView 1.0
+import Sailfish.WebView.Popups 1.0
+
+ApplicationWindow {
+    id: root
+
+    initialPage: Component {
+        Page {
+            id: page
+            WebView {
+                anchors.fill: parent
+                active: true
+                url: "https://browser.sailfishos.org/tests/testpage.html"
+                popupProvider: PopupProvider {
```

Comment:
This starts looking quite neat API now.

Reply:
Yup, it's much nicer than the raw JS dict.  Thanks for the suggestion!